### ISSUE TITLE
[Feature request] Add a window to show all the keyboard shortcuts

### ISSUE THREAD

**Issue by Postroutine** [timbertson/shellshape#203]
Gnome applications start to provide the list of their own keyboard shortcuts. For exemple, Nautilus: https://blogs.gnome.org/aday/files/2015/12/nautilus-shortcuts.png

It would be good if Shellshape can do the same.

**Comment by timbertson** [comment]
I agree, that would be quite nice. If anyone wants to have a go at building this, that would be great (I don't really have the bandwidth for it myself). The main issue is that keybindings are changeable, so it might need to be fancier than you'd expect.
